src/pretalx/mail/models.py

```diff
@@ -162,6 +162,9 @@ def to_mail(
             )
             if commit:
                 mail.save()
+                if "submission" in context_kwargs:
+                    submission = context_kwargs["submission"]
+                    mail.submissions.add(submission)
```

Comment:
We could additionally also add an optional `submissions=None` kwarg to this method, and use that to add passed submissions additionally.

Not sure this is a good idea – `to_mail` already has a long signature, and making it longer isn’t great. But I don’t really see a better way of actually adding multiple submissions here, which would be useful e.g. from the `Schedule.generate_notifications` method, which is (I think) the currently only place where we explicitly send emails concerning multiple proposals, and it would be nice to link those emails to the proposals included. What do you think?